lms/product/plugin/grouping.py

```diff
@@ -45,6 +45,9 @@ def get_sections_for_grading(
 
         return None
 
+    def get_group_sets(self, course_id):  # prama: nocover
+        """Get the group sets available for this course. Used while configuring the assignments."""
```

Comment:
```suggestion
        """Return the list of group sets for the given course."""
```

I think the docstring should stick to describing the contract of the method rather than when the method is used, otherwise it's likely to become out of date. I think just grepping for the method is going to be the only reliable way to find out when it's used